tasks/eva_3454/copyMD5Checksum.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+from urllib.parse import urlsplit
+
+import psycopg2
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+from ebi_eva_internal_pyutils.config_utils import get_properties_from_xml_file
+from ebi_eva_internal_pyutils.pg_utils import get_all_results_for_query, execute_query, logger
+
+logger = log_cfg.get_logger(__name__)
+
+
+def get_contig_alias_db_cred_for_profile(settings_xml_file, profile):
+    properties = get_properties_from_xml_file(profile, settings_xml_file)
+    contig_alias_db = properties['contig-alias.db-url']
+    contig_alias_user = properties['contig-alias.db-username']
+    contig_alias_pass = properties['contig-alias.db-password']
+    return contig_alias_db, contig_alias_user, contig_alias_pass
+
+
+def get_contig_alias_connection_handle(settings_xml_file, profile):
+    pg_url, pg_user, pg_pass = get_contig_alias_db_cred_for_profile(settings_xml_file, profile)
+    return psycopg2.connect(urlsplit(pg_url).path, user=pg_user, password=pg_pass)
+
+
+def get_assemblies_to_update(private_config_xml_file, source_env, target_env, assembly_list):
+    target_asm = set()
+    target_query = f"select distinct assembly_insdc_accession from chromosome"
```

Comment:
That should allow you to run the script multiple time if needed
```suggestion
    target_query = f"select distinct assembly_insdc_accession from chromosome where md5checksum is null"
```

Reply:
updated